app/assets/javascripts/blacklight/blacklight.js

```diff
@@ -69,153 +69,140 @@ Blacklight.onLoad(function () {
       source: terms.ttAdapter()
     });
   });
-});
-
-(function ($) {
-  //change form submit toggle to checkbox
-  Blacklight.doBookmarkToggleBehavior = function () {
-    if (typeof Blacklight.do_bookmark_toggle_behavior == 'function') {
-      console.warn("do_bookmark_toggle_behavior is deprecated. Use doBookmarkToggleBehavior instead.");
-      return Blacklight.do_bookmark_toggle_behavior();
-    }
+}); //change form submit toggle to checkbox
```

Comment:
Note: this is the generated (dist) code, jump to `app/javascript/*` for source changes.